packages/superset-ui-core/src/query/constants.ts

```diff
@@ -17,3 +25,44 @@
  * under the License.
  */
 export const DTTM_ALIAS = '__timestamp';
+
+export const EXTRA_FORM_DATA_OVERRIDE_REGULAR_KEYS: (keyof ExtraFormDataOverrideRegular)[] = [
+  'granularity',
+  'granularity_sqla',
+  'time_column',
+  'time_grain',
+  'time_range',
+];
+
+export const EXTRA_FORM_DATA_OVERRIDE_EXTRA_KEYS: (keyof ExtraFormDataOverrideExtras)[] = [
+  'druid_time_origin',
+  'relative_start',
+  'relative_end',
+  'time_grain_sqla',
+  'time_range_endpoints',
+];
+
+export const EXTRA_FORM_DATA_OVERRIDE_KEYS: (keyof ExtraFormDataOverride)[] = [
+  ...EXTRA_FORM_DATA_OVERRIDE_REGULAR_KEYS,
+  ...EXTRA_FORM_DATA_OVERRIDE_EXTRA_KEYS,
+];
+
+export const EXTRA_FORM_DATA_APPEND_KEYS: (keyof ExtraFormDataAppend)[] = [
+  'adhoc_filters',
+  'filters',
+  'cross_groupby',
+  'cross_highlight',
+  'cross_drilldown',
```

Comment:
what is it property `cross_drilldown`? why we all these properties have prefix `cross`? may be also native filters can do same thing?

Reply:
We discussed it with @amitmiran137 - we just needed to come up with a prefix that clearly distinguishes these from regular form data fields, and `cross` seemed like the most descriptive one we could come up with. `cross_drilldown` is a placeholder for the forthcoming drilldown action.

Reply:
but if native filter will use `cross_groupby` it can cause some misunderstanding, no? may be something more common? also what the difference between `cross_highlight` and `cross_drilldown`?

Reply:
I'm open to another name, but in the interest of time let's decide these asap so we can get this in. @amitmiran137 thoughts?

Reply:
drilldown/highlight are almost the same expect that there indication might look different on the chart.
We mostly gonna use drilldown to completely required and rebuild the visual.
And highlight might just highlight what was filtered instead of rebuilding

Reply:
Does calling it cross_groupby or groupby  matters now that we explicitly support  it as an appendFormData Option ?
If not so we can just call it groupby and then make sure we document it correctly

Reply:
ok, so rename to `groupby`, highlight` and `drilldown`?

Reply:
Hmm, `groupby` will be dangerous, as it's practically a reserved keyword on `QueryFormData`. I'd prefer to prefix these to avoid collisions and to make it obvious that they belong to the same category of actions.

Reply:
Maybe we can consider the following naming : 

filter_highlight
filter_drilldown
groupby